src/servables_config_manager_module/servablesconfigmanagermodule.cpp

```diff
@@ -31,19 +31,27 @@ ServablesConfigManagerModule::ServablesConfigManagerModule() {}
 
 Status ServablesConfigManagerModule::start(const ovms::Config& config) {
     state = ModuleState::STARTED_INITIALIZE;
-    SPDLOG_INFO("{} starting", SERVABLES_CONFIG_MANAGER_MODULE_NAME);
+    SPDLOG_TRACE("{} starting", SERVABLES_CONFIG_MANAGER_MODULE_NAME);
     state = ModuleState::INITIALIZED;
-    SPDLOG_INFO("{} started", SERVABLES_CONFIG_MANAGER_MODULE_NAME);
+    SPDLOG_TRACE("{} started", SERVABLES_CONFIG_MANAGER_MODULE_NAME);
     if (config.getServerSettings().serverMode == LIST_MODELS_MODE) {
         const auto& repositoryPath = config.getServerSettings().hfSettings.downloadPath;
         auto map = listServables(repositoryPath);
         std::stringstream ss;
         for (const auto& [k, v] : map) {
             ss << k << std::endl;
         }
-        SPDLOG_INFO("Available servables to serve from path: {} are:\n{}", repositoryPath, ss.str());
+        std::cout << "Available servables to serve from path: " << repositoryPath << " are: " << std::endl
```

Comment:
There is already existing pr:
https://github.com/openvinotoolkit/model_server/pull/3299/files.
Adding This: "Available servables to serve from path: " makes unnecessarily scripting output of ovms harder